lessons/08_cloud_intro/Week2_README.md

```diff
@@ -0,0 +1,186 @@
+6. Working with the Cloud Shell Editor  
+7. Executing Python in the Cloud  
+8. Connecting with VS Code Desktop (Optional)  
+9. Wrap-Up & Next Steps  
+
+---
+
+## **2.1 Introduction to the Azure Portal**
+
+### **Overview**  
+The Azure Portal is a **web-based dashboard** for managing Azure services. It enables you to:  
+- Create and monitor resources (VMs, storage, databases).  
+- Configure networking and security.  
+- Manage subscriptions and billing.  
+
+### **Real-World Use Cases:**  
+- **Developers:** Deploy and scale applications.  
+- **IT Teams:** Manage identity, access, and security.  
+- **Businesses:** Ensure reliability, compliance, and scalability.  
+
+🔗 [Azure Portal Overview](https://learn.microsoft.com/en-us/azure/azure-portal/azure-portal-overview)  
+
+---
+
+## **2.2 Logging in & Exploring the Portal**
+
+### **Steps:**  
+1. Open [https://portal.azure.com](https://portal.azure.com).  
+2. Sign in with your Microsoft account (work, school, or sandbox).  
+3. Explore the key UI areas:  
+   - **Search bar** — Quickly find any service.  
+   - **Create a resource** — Add new Azure resources.  
+   - **Dashboard** — Customizable home page.  
+   - **Cloud Shell icon** — Access Cloud Shell from the top menu.  
+
+**Activity:** Search for *“Virtual Machine”* and open the service page.  
+
+---
+
+## **2.3 Getting Started with Cloud Shell**
+
+### **Overview**  
+Azure Cloud Shell provides a browser-based command-line interface.  
+
+### **Steps:**  
+1. Click the **Cloud Shell icon** in the portal header.  
+2. Select **Bash** when prompted.  
+3. Accept the prompt to create storage (saves your files) or continue without storage.  
+4. Select your subscription from the drop-down.  
+
+### **Test the Environment:**  
+```bash
+az --version
+python3 --version
+```  
+
+---
+
+## **2.4 Exploring the Cloud Shell Filesystem**
+
+### **Example Commands:**  
+```bash
+pwd                       # show current directory
+ls -la                    # list files
+mkdir azure-lab           # create a folder
+cd azure-lab
+echo "Welcome to Azure Cloud Shell" > hello.txt
+cat hello.txt
+```  
+
+**Outcome:** Students will be comfortable navigating and managing files within the cloud-based filesystem.  
+
+---
+
+## **2.5 Running Azure CLI Commands**
+
+### **Overview**  
+The **Azure CLI** (`az`) is a command-line tool to manage Azure resources programmatically.  
+
+### **Example Commands:**  
+```bash
+az account show --output table     # show current subscription
+az account list --output table     # list all subscriptions
+az group list --output table       # list existing resource groups
+az vm --help                       # get VM-related help
+```  
+
+🔗 [Azure CLI Documentation](https://learn.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest)  
+
+---
+
+## **2.6 Working with the Cloud Shell Editor**
+
+### **Steps:**  
+1. Open the editor in Cloud Shell:  
+   ```bash
+   code .
```

Comment:
See my comments above (my comment number 4). This is a great central part of this whole lesson I love it! That said, we want to provide quite a bit more hand-holding with this. Drop in a screenshot, explain the screens that come up. Where are they entering the different bits of information, etc? Explain like they will have no idea what is going on. What are the different parts of the editor, etc?